### ISSUE TITLE
Not compatible with Peertube v7

### ISSUE THREAD

**Issue by quentinDupont** [McFlat/peertube-plugin-custom-links#14]
Hello and thanks for this addon !

I just upgrade to v7 and this add-on doesn't work anymore with all the (great) design changes.

Do you plan to make it work ?
I can help testing if needed

Thanks !

**Comment by stevespaw** [comment]
Also - we use this plug-in and would love to see it updated to version 7 GUI. 😄

**Comment by DeadSuperHero** [comment]
Yep, also stepping in here. This plugin stopped working for us after the v7 upgrade. I'm curious to see what changes need to be made for compatibility.

**Comment by DeadSuperHero** [comment]
Looks like this might actually only require a minor adjustment. If we look at the [common-client-plugin.js](https://github.com/McFlat/peertube-plugin-custom-links/blob/master/client/common-client-plugin.js), it seems to be targeting a div class that no longer exists, `.top-menu`. It just needs to be changed to `.main-menu`, which is what the new UI makes use of.

**Comment by DeadSuperHero** [comment]
I went ahead and created a pull request in #15, although I'm not really confident that this plugin is being actively maintained.

**Comment by filmfreedom-org** [comment]
As this PR has been sitting for a few months, and I don't see any activity on this plugin, is there a way to fork it into a new plugin we could use, maybe? I'm not really sure how that works. But I was relying heavily on this plugin, and now it doesn't work. It's very sad.

**Comment by DeadSuperHero** [comment]
@filmfreedom-org I actually switched over to the better-menu plugin, it's a good improvement: https://github.com/ritteraf/peertube-plugin-better-menu

You can see what it looks like [on my instance here.](https://spectra.video)

**Comment by filmfreedom-org** [comment]
Yes, you're right. better-menu is doing the job for me, now.
https://tv.filmfreedom.net
